### PR TITLE
Wristbound Computer Introduction

### PR DESCRIPTION
**Pull request by Geevies**
* Tweaked access checking to check for IDs within your gloves slot.
* Tech Support now spawns with a wristbound computer.
* Created a new loadout section just for modular computers.
* "If you have your modular computer, be it a laptop, tablet, or wristbound, in your active hand, ID, or glove slot, it will now allow you use it to access things like airlocks and computer programs.
* Modular tablets now fit in the ID slot.
* Wristbound computers fit in the ID and Gloves slot.
* Fixed security having access to the investigation department's programs.

Wrist computer sprites ported from Europa. Program/screen sprites by Aticius and slightly edited by me.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by MistakeNot4892** [comment]
These are the Europa wrist computer icons, not by Atticus.

File: code/game/jobs/access.dm [hunk 1]
```diff
@@ -243,7 +243,13 @@ var/obj/item/card/id/all_access/ghost_all_access
 		if(id)
 			return id
 	if(wear_id)
-		return wear_id.GetID()
+		var/id = wear_id.GetID()
+		if(id)
+			return id
```
Comment (NonQueueingMatt) [inline code/game/jobs/access.dm:248]: This is the same thing in more steps.

File: code/game/jobs/access.dm [hunk 2]
```diff
@@ -243,7 +243,13 @@ var/obj/item/card/id/all_access/ghost_all_access
...
+	if(gloves)
+		var/id = gloves.GetID()
+		if(id)
+			return id
```
Comment (NonQueueingMatt) [inline code/game/jobs/access.dm:252]: See above.

File: code/modules/modular_computers/computers/modular_computer/interaction.dm [hunk 3]
```diff
@@ -224,3 +224,9 @@
 		return
 
 	..()
+
+/obj/item/modular_computer/GetID()
+	if(!card_slot)
+		return null
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/computers/modular_computer/interaction.dm:230]: You don't need this.

File: html/changelogs/geeves-wristbound_computers.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+author: Code by Geeves, Sprites by Aticius & Europa Devs
```
Comment (NonQueueingMatt) [inline html/changelogs/geeves-wristbound_computers.yml:1]: What sprites are by aticius?

Comment (Geevies) [reply html/changelogs/geeves-wristbound_computers.yml:1]: the programs

File: code/game/jobs/access.dm (hunk 1 above)
Comment (Geevies) [reply code/game/jobs/access.dm:248]: It checks whether it has an ID. If it doesn't, it checks the next one. Returning early means the proc stops without checking the rest.

File: code/game/jobs/access.dm (hunk 2 above)
Comment (Geevies) [reply code/game/jobs/access.dm:252]: See above.

**Pull request merged**